make/test.mk

```diff
@@ -56,6 +56,32 @@ e2e-cleanup: get-test-namespace
 	$(Q)-oc delete -f ./deploy/test/role_binding_test.yaml --namespace $(TEST_NAMESPACE)
 	$(Q)-oc delete -f ./deploy/test/operator_test.yaml --namespace $(TEST_NAMESPACE)
 
+.PHONY: test-olm-integration
+## Runs the OLM integration tests without coverage
+test-olm-integration: push-operator-image olm-integration-setup
```

Comment:
This calls 
```
.PHONY: build-operator-image
## Build and create the operator container image
build-operator-image:
	operator-sdk build $(DEVCONSOLE_OPERATOR_IMAGE)
```

The CI test pushing to the `latest` tag can be disastrous :)

Reply:
Fixed. d27da007c745d079d942540e67f2caccfe4b7034